ui/mirage/factories/token.js

```diff
@@ -41,7 +41,7 @@ export default Factory.extend({
       // Create a special policy with variables rules in place
       if (token.id === '53cur3-v4r14bl35') {
         const variableMakerPolicy = {
-          id: 'Variable Maker',
+          id: 'Variable-Maker',
```

Comment:
why the name change?

Reply:
The new section allows us to create and edit policy names, and something that was added was a regex name check. Because policies' names are their IDs, spaces (and non-alphanumeric + dash characters) are not allowed. This fixes it and lets us, in the mocked version, save this particular policy (and variable-reader)